### DISCUSSION TITLE
PHP/Symfony

### DISCUSSION THREAD

**Discussion by Mohamedaymen2020** [symfony/symfony#42475] [Q&A] [answered]
Hello guys i'am new in php and in symfony ; actually i have a php code written with symfony 3.0.1 and i want to run it with symfony 4.2 , what the steps that i have to follow , thank you

**Comment by linaori** [comment, marked as answer]
Long story short: Go to 3.4 first, fix deprecations, then go to 4.4. You can then go to the latest 5.x. 

You can find the upgrade documentation here:
 - https://symfony.com/doc/current/setup/upgrade_minor.html
 - https://symfony.com/doc/current/setup/upgrade_major.html
